lib/class-wp-rest-dependencies-controller.php

```diff
@@ -98,6 +100,9 @@ public function get_items( $request ) {
 
 		if ( $handle ) {
 			foreach ( $filter as $dependency_handle ) {
+				if ( $handle === $dependency_handle ) {
```

Comment:
👍

Reply:
Can we add a unit test for this?

Reply:
@spacedmonkey I've added dedicated test for it, even it above solution was covered by `get_items` tests (like checks of number of dependencies returned etc).